api/prisma/schema.prisma

```diff
@@ -9,3 +9,21 @@ datasource db {
   provider = "mysql"
   url      = env("DATABASE_URL")
 }
+
+model Deal {
+  dealId Int @id @default(autoincrement())
+  business @relation(fields: [dealId], references: [businessId])
```

Comment:
business should be:
`business @relation(fields: [businessId], references: [businessId])`